auth/src/com/sixsq/slipstream/auth/internal.clj

```diff
@@ -10,13 +10,20 @@
   (:import (java.util Date TimeZone)
            (java.text SimpleDateFormat)))
 
+(defn- xor
+  [a b]
+  (and (not (and a b))
+       (or a b)))
+
 (defn- extract-credentials
   [request]
-  (uh/select-in-params request [:user-name :password]))
+  (let [username (->> request :params ((juxt :user-name :username)) (apply xor))]
```

Comment:
Supporting `user-name` seems reasonable to me.
Yes, there is a test checking inconsistent parameters:
https://github.com/slipstream/SlipStreamServer/pull/727/files/3e832a3edcfb84e5456687a483dc169efa23b599#diff-18000da1b1f36b867e0a4cd103ae1df3R22